app/models/ability.rb

```diff
@@ -0,0 +1,16 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(member)
+    member ||= Member.new
+    if member
```

Comment:
asta e o prostie pentru ca o sa ai mereu member aici. Pentru ca tu faci Member.new in caz ca nu e definit. E stupid. Sterge `member ||= Member.new` din prima linie ca altfel o sa mearga intodeauna asta... Si de fapt ability ast anu iti trebuie deloc nu ii vad rostu.